config/routes.rb

```diff
@@ -13,6 +13,6 @@
   namespace :admin do
     get "/", to: "base#index", as: "main_page"
     resources :users, only: [:index]
-    resources :information_architectures, only: [:index]
+    resources :information_architectures, only: %i[index create]
```

Comment:
create
`resources :categories `
inside arch ...
to have route 
POST skdskdk.com/admin/information_arch/categories